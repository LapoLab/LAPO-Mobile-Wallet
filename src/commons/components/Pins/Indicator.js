/**
 * @flow
 */

import React, { Component } from 'react';
import { View } from 'react-native';
import styles from './styles';

type Props = {
  pins: string,
};
class Indicator extends Component<Props> {
  render() {
    const { pins } = this.props;
    const fillPinsLength = pins && pins.length ? pins.length : 0;
    const emptyPinsLength = 6 - fillPinsLength;
    return (
      <View style={styles.containerIndicator}>
        <View style={styles.rowIndicator}>
          {fillPinsLength > 0
            && fillPinsLength < 7
            && Array(fillPinsLength)
              .fill()
              .map((item, index) => (
                <View
                  style={styles.indicatorFillCircle}
                  // eslint-disable-next-line
                  key={`fill-${index}`}
                />
              ))}
          {emptyPinsLength > 0
            && emptyPinsLength < 7
            && Array(emptyPinsLength)
              .fill()
              .map((item, index) => (
                <View
                  style={styles.indicatorEmptyCircle}
                  // eslint-disable-next-line
                  key={`empty-${index}`}
                />
              ))}
        </View>
      </View>
    );
  }
}

export default Indicator;
