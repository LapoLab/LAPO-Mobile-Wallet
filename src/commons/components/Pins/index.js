/**
 * @flow
 */

import React, { Component } from 'react';
import {
  Text,
  View,
  TouchableOpacity,
} from 'react-native';
import styles from './styles';
import Indicator from './Indicator';

type Props = {
  pins: string,
  onChange: Function,
};
class Pins extends Component<Props> {
  handleChoose = (nameChoose: string) => () => {
    const { onChange, pins } = this.props;
    if (pins.length < 7) {
      onChange(nameChoose);
    }
  };

  render() {
    const { pins } = this.props;
    return (
      <View style={styles.container}>
        <Indicator pins={pins} />

        <View style={styles.row}>
          <View style={styles.item}>
            <TouchableOpacity style={styles.outerButton} onPress={this.handleChoose('1')}>
              <View style={styles.button}>
                <Text style={styles.textButton}>1</Text>
              </View>
            </TouchableOpacity>
          </View>
          <View style={styles.item}>
            <TouchableOpacity style={styles.outerButton} onPress={this.handleChoose('2')}>
              <View style={styles.button}>
                <Text style={styles.textButton}>2</Text>
              </View>
            </TouchableOpacity>
          </View>
          <View style={styles.item}>
            <TouchableOpacity style={styles.outerButton} onPress={this.handleChoose('3')}>
              <View style={styles.button}>
                <Text style={styles.textButton}>3</Text>
              </View>
            </TouchableOpacity>
          </View>
        </View>

        <View style={styles.row}>
          <View style={styles.item}>
            <TouchableOpacity style={styles.outerButton} onPress={this.handleChoose('4')}>
              <View style={styles.button}>
                <Text style={styles.textButton}>4</Text>
              </View>
            </TouchableOpacity>
          </View>
          <View style={styles.item}>
            <TouchableOpacity style={styles.outerButton} onPress={this.handleChoose('5')}>
              <View style={styles.button}>
                <Text style={styles.textButton}>5</Text>
              </View>
            </TouchableOpacity>
          </View>
          <View style={styles.item}>
            <TouchableOpacity style={styles.outerButton} onPress={this.handleChoose('6')}>
              <View style={styles.button}>
                <Text style={styles.textButton}>6</Text>
              </View>
            </TouchableOpacity>
          </View>
        </View>

        <View style={styles.row}>
          <View style={styles.item}>
            <TouchableOpacity style={styles.outerButton} onPress={this.handleChoose('7')}>
              <View style={styles.button}>
                <Text style={styles.textButton}>7</Text>
              </View>
            </TouchableOpacity>
          </View>
          <View style={styles.item}>
            <TouchableOpacity style={styles.outerButton} onPress={this.handleChoose('8')}>
              <View style={styles.button}>
                <Text style={styles.textButton}>8</Text>
              </View>
            </TouchableOpacity>
          </View>
          <View style={styles.item}>
            <TouchableOpacity style={styles.outerButton} onPress={this.handleChoose('9')}>
              <View style={styles.button}>
                <Text style={styles.textButton}>9</Text>
              </View>
            </TouchableOpacity>
          </View>
        </View>

        <View style={styles.row}>
          <View style={styles.buttonEmpty} />
          <View>
            <TouchableOpacity style={styles.outerButton} onPress={this.handleChoose('0')}>
              <View style={styles.button}>
                <Text style={styles.textButton}>0</Text>
              </View>
            </TouchableOpacity>
          </View>
          <View>
            <TouchableOpacity style={styles.buttonEmpty} onPress={this.handleChoose('delete')}>
              <Text style={styles.textButtonDelete}>DELETE</Text>
            </TouchableOpacity>
          </View>
        </View>
      </View>
    );
  }
}

export default Pins;
