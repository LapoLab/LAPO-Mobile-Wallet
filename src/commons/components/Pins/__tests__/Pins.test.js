// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import Pins from '../index';

describe('Pins component', () => {
  test('renders snapshots default', () => {
    const fn = jest.fn();
    const tree = renderer.create(<Pins pins="" onChange={fn} />); // eslint-disable-line
    const render = tree.toJSON();
    expect(render).toMatchSnapshot();
  });

  test('renders snapshots with value', () => {
    const fn = jest.fn();
    const tree = renderer.create(<Pins pins="12333212333s" onChange={fn}  />); // eslint-disable-line
    const render = tree.toJSON();
    expect(render).toMatchSnapshot();
  });
});
