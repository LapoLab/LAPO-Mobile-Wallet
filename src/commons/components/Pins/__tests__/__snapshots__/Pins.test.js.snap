// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pins component renders snapshots default 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "column",
      "paddingBottom": 10,
      "paddingHorizontal": "10%",
      "paddingdVertical": 0,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "height": 40,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "space-around",
          "paddingVertical": 10,
          "width": 180,
        }
      }
    >
      <View
        style={
          Object {
            "borderColor": "#4183BD",
            "borderRadius": 6,
            "borderWidth": 1,
            "height": 12,
            "width": 12,
          }
        }
      />
      <View
        style={
          Object {
            "borderColor": "#4183BD",
            "borderRadius": 6,
            "borderWidth": 1,
            "height": 12,
            "width": 12,
          }
        }
      />
      <View
        style={
          Object {
            "borderColor": "#4183BD",
            "borderRadius": 6,
            "borderWidth": 1,
            "height": 12,
            "width": 12,
          }
        }
      />
      <View
        style={
          Object {
            "borderColor": "#4183BD",
            "borderRadius": 6,
            "borderWidth": 1,
            "height": 12,
            "width": 12,
          }
        }
      />
      <View
        style={
          Object {
            "borderColor": "#4183BD",
            "borderRadius": 6,
            "borderWidth": 1,
            "height": 12,
            "width": 12,
          }
        }
      />
      <View
        style={
          Object {
            "borderColor": "#4183BD",
            "borderRadius": 6,
            "borderWidth": 1,
            "height": 12,
            "width": 12,
          }
        }
      />
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(93,112,241, 0.2)",
            "borderRadius": 48,
            "height": 96,
            "justifyContent": "center",
            "opacity": 1,
            "width": 96,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#4183BD",
              "borderRadius": 40,
              "height": 80,
              "justifyContent": "center",
              "width": 80,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
                "fontFamily": "Lato-Bold",
                "fontSize": 20,
              }
            }
          >
            1
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(93,112,241, 0.2)",
            "borderRadius": 48,
            "height": 96,
            "justifyContent": "center",
            "opacity": 1,
            "width": 96,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#4183BD",
              "borderRadius": 40,
              "height": 80,
              "justifyContent": "center",
              "width": 80,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
                "fontFamily": "Lato-Bold",
                "fontSize": 20,
              }
            }
          >
            2
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(93,112,241, 0.2)",
            "borderRadius": 48,
            "height": 96,
            "justifyContent": "center",
            "opacity": 1,
            "width": 96,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#4183BD",
              "borderRadius": 40,
              "height": 80,
              "justifyContent": "center",
              "width": 80,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
                "fontFamily": "Lato-Bold",
                "fontSize": 20,
              }
            }
          >
            3
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(93,112,241, 0.2)",
            "borderRadius": 48,
            "height": 96,
            "justifyContent": "center",
            "opacity": 1,
            "width": 96,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#4183BD",
              "borderRadius": 40,
              "height": 80,
              "justifyContent": "center",
              "width": 80,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
                "fontFamily": "Lato-Bold",
                "fontSize": 20,
              }
            }
          >
            4
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(93,112,241, 0.2)",
            "borderRadius": 48,
            "height": 96,
            "justifyContent": "center",
            "opacity": 1,
            "width": 96,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#4183BD",
              "borderRadius": 40,
              "height": 80,
              "justifyContent": "center",
              "width": 80,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
                "fontFamily": "Lato-Bold",
                "fontSize": 20,
              }
            }
          >
            5
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(93,112,241, 0.2)",
            "borderRadius": 48,
            "height": 96,
            "justifyContent": "center",
            "opacity": 1,
            "width": 96,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#4183BD",
              "borderRadius": 40,
              "height": 80,
              "justifyContent": "center",
              "width": 80,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
                "fontFamily": "Lato-Bold",
                "fontSize": 20,
              }
            }
          >
            6
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(93,112,241, 0.2)",
            "borderRadius": 48,
            "height": 96,
            "justifyContent": "center",
            "opacity": 1,
            "width": 96,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#4183BD",
              "borderRadius": 40,
              "height": 80,
              "justifyContent": "center",
              "width": 80,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
                "fontFamily": "Lato-Bold",
                "fontSize": 20,
              }
            }
          >
            7
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(93,112,241, 0.2)",
            "borderRadius": 48,
            "height": 96,
            "justifyContent": "center",
            "opacity": 1,
            "width": 96,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#4183BD",
              "borderRadius": 40,
              "height": 80,
              "justifyContent": "center",
              "width": 80,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
                "fontFamily": "Lato-Bold",
                "fontSize": 20,
              }
            }
          >
            8
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(93,112,241, 0.2)",
            "borderRadius": 48,
            "height": 96,
            "justifyContent": "center",
            "opacity": 1,
            "width": 96,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#4183BD",
              "borderRadius": 40,
              "height": 80,
              "justifyContent": "center",
              "width": 80,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
                "fontFamily": "Lato-Bold",
                "fontSize": 20,
              }
            }
          >
            9
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "paddingVertical": 20,
          "textAlign": "center",
          "width": 150,
        }
      }
    />
    <View>
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(93,112,241, 0.2)",
            "borderRadius": 48,
            "height": 96,
            "justifyContent": "center",
            "opacity": 1,
            "width": 96,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#4183BD",
              "borderRadius": 40,
              "height": 80,
              "justifyContent": "center",
              "width": 80,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
                "fontFamily": "Lato-Bold",
                "fontSize": 20,
              }
            }
          >
            0
          </Text>
        </View>
      </View>
    </View>
    <View>
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "justifyContent": "center",
            "opacity": 1,
            "paddingVertical": 20,
            "textAlign": "center",
            "width": 150,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#D5D9F3",
              "fontFamily": "Lato-Regular",
              "fontSize": 14,
              "marginRight": 40,
            }
          }
        >
          DELETE
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`Pins component renders snapshots with value 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "column",
      "paddingBottom": 10,
      "paddingHorizontal": "10%",
      "paddingdVertical": 0,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "height": 40,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "space-around",
          "paddingVertical": 10,
          "width": 180,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(93,112,241, 0.2)",
            "borderRadius": 48,
            "height": 96,
            "justifyContent": "center",
            "opacity": 1,
            "width": 96,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#4183BD",
              "borderRadius": 40,
              "height": 80,
              "justifyContent": "center",
              "width": 80,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
                "fontFamily": "Lato-Bold",
                "fontSize": 20,
              }
            }
          >
            1
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(93,112,241, 0.2)",
            "borderRadius": 48,
            "height": 96,
            "justifyContent": "center",
            "opacity": 1,
            "width": 96,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#4183BD",
              "borderRadius": 40,
              "height": 80,
              "justifyContent": "center",
              "width": 80,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
                "fontFamily": "Lato-Bold",
                "fontSize": 20,
              }
            }
          >
            2
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(93,112,241, 0.2)",
            "borderRadius": 48,
            "height": 96,
            "justifyContent": "center",
            "opacity": 1,
            "width": 96,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#4183BD",
              "borderRadius": 40,
              "height": 80,
              "justifyContent": "center",
              "width": 80,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
                "fontFamily": "Lato-Bold",
                "fontSize": 20,
              }
            }
          >
            3
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(93,112,241, 0.2)",
            "borderRadius": 48,
            "height": 96,
            "justifyContent": "center",
            "opacity": 1,
            "width": 96,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#4183BD",
              "borderRadius": 40,
              "height": 80,
              "justifyContent": "center",
              "width": 80,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
                "fontFamily": "Lato-Bold",
                "fontSize": 20,
              }
            }
          >
            4
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(93,112,241, 0.2)",
            "borderRadius": 48,
            "height": 96,
            "justifyContent": "center",
            "opacity": 1,
            "width": 96,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#4183BD",
              "borderRadius": 40,
              "height": 80,
              "justifyContent": "center",
              "width": 80,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
                "fontFamily": "Lato-Bold",
                "fontSize": 20,
              }
            }
          >
            5
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(93,112,241, 0.2)",
            "borderRadius": 48,
            "height": 96,
            "justifyContent": "center",
            "opacity": 1,
            "width": 96,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#4183BD",
              "borderRadius": 40,
              "height": 80,
              "justifyContent": "center",
              "width": 80,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
                "fontFamily": "Lato-Bold",
                "fontSize": 20,
              }
            }
          >
            6
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(93,112,241, 0.2)",
            "borderRadius": 48,
            "height": 96,
            "justifyContent": "center",
            "opacity": 1,
            "width": 96,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#4183BD",
              "borderRadius": 40,
              "height": 80,
              "justifyContent": "center",
              "width": 80,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
                "fontFamily": "Lato-Bold",
                "fontSize": 20,
              }
            }
          >
            7
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(93,112,241, 0.2)",
            "borderRadius": 48,
            "height": 96,
            "justifyContent": "center",
            "opacity": 1,
            "width": 96,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#4183BD",
              "borderRadius": 40,
              "height": 80,
              "justifyContent": "center",
              "width": 80,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
                "fontFamily": "Lato-Bold",
                "fontSize": 20,
              }
            }
          >
            8
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(93,112,241, 0.2)",
            "borderRadius": 48,
            "height": 96,
            "justifyContent": "center",
            "opacity": 1,
            "width": 96,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#4183BD",
              "borderRadius": 40,
              "height": 80,
              "justifyContent": "center",
              "width": 80,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
                "fontFamily": "Lato-Bold",
                "fontSize": 20,
              }
            }
          >
            9
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "paddingVertical": 20,
          "textAlign": "center",
          "width": 150,
        }
      }
    />
    <View>
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(93,112,241, 0.2)",
            "borderRadius": 48,
            "height": 96,
            "justifyContent": "center",
            "opacity": 1,
            "width": 96,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#4183BD",
              "borderRadius": 40,
              "height": 80,
              "justifyContent": "center",
              "width": 80,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
                "fontFamily": "Lato-Bold",
                "fontSize": 20,
              }
            }
          >
            0
          </Text>
        </View>
      </View>
    </View>
    <View>
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "justifyContent": "center",
            "opacity": 1,
            "paddingVertical": 20,
            "textAlign": "center",
            "width": 150,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#D5D9F3",
              "fontFamily": "Lato-Regular",
              "fontSize": 14,
              "marginRight": 40,
            }
          }
        >
          DELETE
        </Text>
      </View>
    </View>
  </View>
</View>
`;
